.github/workflows/notify_team_new_comment.yml

```diff
@@ -0,0 +1,27 @@
+name: Send a slack notification when a contributor comments on issue
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  contributor_issue_comment:
+    name: Contributor issue comment
+
+    if: >-
+      ${{
+        !github.event.issue.pull_request &&
+        github.event.comment.author_association != 'MEMBER' &&
+        github.event.comment.author_association != 'OWNER'
+      }}
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send message to Slack channel
+        env:
+            SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+        run: |
+          curl -X POST -H 'Content-type: application/json' \
+          --data \
+          '{"text": "New comment on issue: '${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/issues/${{ github.event.issue.number }}' by '${GITHUB_ACTOR}'"}' \
```

Comment:
Possible tweaks here:
* Put the issue name instead of just 'issue'
* Make a link out of the issue name
* Update the link to include the reference to the specific comment so that clicking on it will open the issue at that specific comment
* Could also make the github actor text link to the user profile of the github user.

Reply:
I've included the issue title and linked it to the specific comment. I have not added github profile user link because that would be unnecessary.